### ISSUE TITLE
Regarding the closed source

### ISSUE THREAD

**Issue by LocalsGitHub** [Rdimo/Hazard-Nuker#10]
Even though this program is closed source, it is still decompilable which I did. I will not be releasing the source code in respect of the authors wish to keep it closed. I can verify/vouch that it is clean.

**Comment by Rdimo** [comment]
yes thank you for respecting that, altough I will maybe release the source code soon either way since ye, why not
